packages/browser-wallet/src/popup/pages/Identity/Identity.scss

```diff
@@ -0,0 +1,32 @@
+.identity {
+    &__attributes {
```

Comment:
Add some margin after the attributes so that the attribute box does not touch the bottom of the window.